styles/anilist/catppuccin.user.css

```diff
@@ -25,6 +25,16 @@
     #catppuccin(@darkFlavor, @accentColor);
   }
 
+  // Scrollbar color
+  * {
+    scrollbar-color: @catppuccin[@@lightFlavor][@@accentColor]
+      fade(@catppuccin[@@darkFlavor][@crust], 0.2);
+    &:has(.site-theme-dark) {
+      scrollbar-color: @catppuccin[@@darkFlavor][@@accentColor]
+        fade(@catppuccin[@@darkFlavor][@crust], 0.2);
+    }
+  }
+
```

Comment:
I'm not completely sure if this will be fine performance-wise, but it's the only way I could style the scrollbars.

Reply:
Why does it need to be applied to every single element? And why is this not in the #catppuccin mixin?

Reply:
The userstyle uses the body element to select dark and light flavors, which unfortunately results in the scrollbar (Which is styled using the `*` selector) not being themed. It's the only way I could figure out a workaround, however I'm open to any changes.